cpp/src/parquet/bloom_filter_benchmark.cc

```diff
@@ -0,0 +1,148 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "benchmark/benchmark.h"
+
+#include "parquet/bloom_filter.h"
+#include "parquet/test_util.h"
+
+namespace parquet {
+namespace benchmark {
+
+template <typename DType>
+static void BM_CountHash(::benchmark::State& state) {
+  using T = typename DType::c_type;
+  std::vector<T> values(1024);
```

Comment:
Please avoid magic numbers and use constants instead.
```suggestion
  constexpr int kNumValues = 1024;
  std::vector<T> values(1024);
```